gatsby-config.js

```diff
@@ -0,0 +1,63 @@
+module.exports = {
+  siteMetadata: {
+    title: `SilverStripe Documentation`,
+    description: `Developer documentation for the SilverStripe CMS and framework.`,
+    author: `The Silverstripe Community`
+  },
+  plugins: [
+    `gatsby-plugin-react-helmet`,
+    `gatsby-plugin-sass`,
+    {
+      resolve: `gatsby-source-filesystem`,
+      options: {
+        name: `files`,
+        path: `${__dirname}/static`
+      }
+    },    
+    {
+      resolve: `gatsby-source-git`,
+      options: {
+        name: `4`,
+        remote: `https://github.com/unclecheese/silverstripe-framework.git`,
+        branch: `pulls/4/gatsby-docs`,
```

Comment:
Needs branch retargeting?

Reply:
Yeah, but as we discussed yesterday, not critical for go-live.